Misc/NEWS.d/next/Core and Builtins/2021-08-09-19-05-20.bpo-44874.oOcfU4.rst

```diff
@@ -0,0 +1,38 @@
+Deprecate the old trashcan macros (``Py_TRASHCAN_SAFE_BEGIN``/``Py_TRASHCAN_SAFE_END``). They should be replaced by the new macros ``Py_TRASHCAN_BEGIN`` and ``Py_TRASHCAN_END``.
```

Comment:
Few people read the Changelog. I suggest to move these explanations to What's New in Python 3.11 > C API > Porting to Python 3.11:
https://docs.python.org/dev/whatsnew/3.11.html#id3

Reply:
I'll add that.